src/store/actions/authActions.js

```diff
@@ -97,35 +97,47 @@ export const signUp = (newUser) => {
           firebase
             .auth()
             .createUserWithEmailAndPassword(newUser.email, newUser.password)
-            .then((response) => {
+            .then(async (response) => {
```

Comment:
This does not have to be `async`.
```suggestion
            .then((response) => {
```